lib/takeaway.rb

```diff
@@ -0,0 +1,66 @@
+class Takeaway
+
+  MENU = { "Spring Roll" => 1.95,
+           "Prawn Crackers" => 1.60,
+           "Fried Pork Dumpling" => 2.10,
+           "Won Ton" => 3.70,
+           "Chicken and Noodle Soup" => 1.90,
+           "Pork Chow Mein" => 4.60,
+           "Szechuan Vegetables" => 4.15,
+           "King Prawn Kung Po" => 5.40
+         }.freeze
+
+  def delivery_calc
+    time = (Time.now + 60 * 60).asctime.split[3]
+  end
+
+  def price_calc(price, number = 1)
+    @total += price * number
+  end
+
+  def initialize(menu: MENU)
+    @basket = []
+    @menu = menu
+    @total = 0
+    @basket_summary = []
+  end
+
+  def menu
+    MENU
+  end
+
+  def basket_summary
```

Comment:
Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. (https://github.com/bbatsov/ruby-style-guide#attr_family)